libvast/vast/logger.hpp

```diff
@@ -68,9 +68,9 @@ auto id_or_name(T&& x) {
       return caf::detail::pretty_type_name(typeid(value_type));
   } else {
     if constexpr (has_ostream_operator<T>)
-      return x;
+      return std::forward<T&&>(x);
```

Comment:
Should this be `std::forward<T>` as opposed to `T&&`?

Reply:
duh. Thanks for paying attention.